org.eclipse.winery.workflowmodeler/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "workflow-modeler",
+  "version": "1.0.0",
+  "description": "A Designer For workflow",
+  "scripts": {
+    "start": "webpack-dev-server --inline --progress",
+    "test": "karma start",
+    "build": "rimraf dist && webpack --config config/webpack.prod.js --progress --profile --bail",
+    "postinstall": "typings install",
+    "mock": "node mock/server.js",
+    "lint": "tslint ./src/**/*.ts -t verbose"
+  },
+  "licenses": [
+    {
+      "type": "Apache-2.0",
+      "url": "https://github.com/ZTE/winery/blob/workflow-modeler/org.eclipse.winery.workflowmodeler/LICENSE-ASL.txt"
+    },
+	{
+      "type": "Eclipse Public License-1.0",
+      "url": "https://github.com/ZTE/winery/blob/workflow-modeler/org.eclipse.winery.workflowmodeler/LICENSE-EPL.txt"
+    }
+  ],
+  "dependencies": {
+    "@angular/common": "2.4.10",
```

Comment:
I would recommend to update angular to the latest version (4.0.1) because of better performance and more consistency. It is also fully backward compatible, so that this should be easy to update (I did it myself some weeks ago for the new winery angular ui).